src/WP_CLI/CommandWithTranslation.php

```diff
@@ -135,6 +135,10 @@ public function install( $args, $assoc_args ) {
 
 		$language_codes = $args;
 
+		if(  1 < count( $language_codes )  &&  in_array( true , $assoc_args ) ){
```

Comment:
Prefer to use `in_array()` in strict mode (third argument set to `true`), otherwise it can have unexpected behavior.

See http://php.net/manual/en/function.in-array.php